src/reference.ts

```diff
@@ -1,6 +1,23 @@
 import Span from './span';
 import SpanContext from './span_context';
 
+const getContext = (contextOrSpan: SpanContext | Span): SpanContext => {
+    if (contextOrSpan instanceof SpanContext) {
+      return contextOrSpan;
+    }
```

Comment:
This doesn't seem necessary.

Reply:
This covers a rare, but possible case when a class, extending SpanContext, has its own "context" method for some reason. Do you think it's not worth covering?